contracts/tokens/Xcert.sol

```diff
@@ -59,28 +74,37 @@ contract Xcert is NFTokenEnumerable, NFTokenMetadata {
     public
   {
     nftConvention = _convention;
-    supportedInterfaces[0x355d09e9] = true; // Xcert
+    supportedInterfaces[0xe353dea7] = true; // Xcert
   }
 
-  /*
+  /**
    * @dev Mints a new NFT.
+   * @notice _dataKey and _dataValue length has to be the same.
    * @param _to The address that will own the minted NFT.
    * @param _id The NFT to be minted by the msg.sender.
+   * @param _uri An URI pointing to NFT metadata.
    * @param _proof Cryptographic asset imprint.
-   * @param _uri An URI pointing to NFT metadata (optional, max length 2083).
+   * @param _config Array of config values.
```

Comment:
"array of internal / protocol config values"? we should be as descriptive as possible here